euth/users/views.py

```diff
@@ -9,4 +11,15 @@ class ProfileView(DetailView):
 
     @property
     def get_participated_projects(self):
-        return Project.objects.filter(participants=self.object)
+        user = self.object
+
+        qs = Project.objects.filter(
+            django_models.Q(
+                module__phase__end_date__gt=timezone.now()
+            ),  # only active projects
+            django_models.Q(follow__creator=user) |
```

Comment:
You have to filter for enabled=True also, otherwise the project will not be removed from the list when you unfollow and reload.